templates/includes/menu.html.twig

```diff
@@ -6,7 +6,7 @@
     {% for item in menu %}
         <li id="{{ item.identifier }}" class="nav-item {{ item.isActive ? 'active' : '' }} {{ item.hasChildren? 'dropdown' : '' }}">
             <a {% if item.hasChildren -%}
-                class="nav-link dropdown-toggle" href="#navbar-base" data-bs-toggle="dropdown" role="button" aria-expanded="false"
+                class="nav-link dropdown-toggle {{ item.hasActiveChildren? 'show' : '' }}" href="#navbar-base" data-bs-toggle="dropdown" role="button" aria-expanded="{{ item.hasActiveChildren? 'true' : 'false' }}"
```

Comment:
is this triggered in both layouts?

Reply:
Not it's not.
Only on horizontal, but on that branch/PR I'm missing the code to condition the if horizontal/vertical.
I can close that PR and put the code on https://github.com/kevinpapst/TablerBundle/pull/3